stable/drupal/templates/drupal-pvc.yaml

```diff
@@ -8,13 +8,18 @@ metadata:
     chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
     release: "{{ .Release.Name }}"
     heritage: "{{ .Release.Service }}"
+  {{- if (not .Values.persistence.drupal.hostPath) }}
   annotations:
   {{- if .Values.persistence.drupal.storageClass }}
     volume.beta.kubernetes.io/storage-class: {{ .Values.persistence.drupal.storageClass | quote }}
   {{- else }}
     volume.alpha.kubernetes.io/storage-class: default
   {{- end }}
+  {{- end }}
 spec:
+  {{- if .Values.persistence.drupal.hostPath }}
+  storageClassName: ""
```

Comment:
Just a note - I found out in https://github.com/kubernetes/kubernetes.github.io/pull/3979 that this is due to k8s using the `default` storage class is one isn't defined, which is why you get a dynamic PV if you don't define this. The PV you create has no storage class, so that's why it isn't picked up. If you define a storage class `manual` for example (it can be anything) on both the PV and PVC it should bind them as you expect.

Not saying we need to change it, just wanted to explain what I learned when looking at that doc issue :)